docker-compose.yml

```diff
@@ -12,6 +12,22 @@ services:
     expose:
       - "34000"
 
+  smtp:
+    hostname: smtp
+    container_name: dev_smtp
+    image: maildev/maildev:2.0.5
+    restart: on-failure
+    ports:
+      - "25:25" # smtp server
+      - "35000:1080" # web ui
```

Comment:
I think that exposing port 25 in your host is not necessary, unless you want to use your laptop as a mail server from outside connections. On the other hand, exposing the web ui port is necessary to use the browser to navigate the ui

Reply:
fixed